workflow/rocoto/gefs_tasks.py

```diff
@@ -324,10 +324,12 @@ def atmos_ensstat(self):
 
     def wavepostsbs(self):
         deps = []
+        dtg_prefix = "@Y@m@d.@H0000"
+        offset = str(self._configs['base']['OFFSET_START_HOUR']).zfill(2) + ":00:00"
```

Comment:
Use `aerosol_init` from `gfs_tasks.py` as a model.
```suggestion
        interval = self._configs['base']['OFFSET_START_HOUR']
        offset = timedelta_to_HMS(interval)
```